configTestWorthReading.xml

```diff
@@ -3,9 +3,10 @@
     <params>
         <searchFile>test/searchWorthReading.html</searchFile>
         <versionFile>test/VERSION</versionFile>
-        <stemmerFolder>en</stemmerFolder>
+        <stemmerFolder>identity</stemmerFolder>
```

Comment:
@martindholmes : Did we want the configTestWorthReading to have the identity stemmer by default? (I have no problem with that being the case, just double-checking)

Reply:
I think we probably should have something in the tests using the identity stemmer. Of course the JS tests will need to be rewritten to match the results we expect from the identity stemmer, which I've neglected to do. I can try and do that tomorrow, and then add those changes to the pull request. Thanks for catching that.